### PR TITLE
Remove deprecated functions. Also, a fix on adjustedPoint method

### PR DESCRIPTION
**Pull request by adjamhub**

### CODE REVIEW

File: lxqtrotatedwidget.cpp
Comment (tsujan) [inline lxqtrotatedwidget.cpp:276]: `adjustedPoint()` is a public method. Changing it may break some code.

Instead, if the wheel event is rewritten as:

```c++
#if (QT_VERSION >= QT_VERSION_CHECK(5,14,0))
    QWheelEvent contentEvent(adjustedPoint(event->position().toPoint()), event->globalPosition(), event->pixelDelta(), event->angleDelta(), event->buttons(), event->modifiers(), event->phase(), false);
#else
.....
```
the compiler warning should be silenced.

Comment (adjamhub) [reply lxqtrotatedwidget.cpp:276]: It's not just because of the compiler warning. adjustedPoint method clearly calculates a QPointF result. And every ctor there (QMouseEvent, QWheelEvent) neets a QPointF.
IMHO the code is better this way. And I could find another call at adjustedPoint() in all my lxqt repos. 
Can you please reconsider? 
Waiting for the response :)

Comment (tsujan) [reply lxqtrotatedwidget.cpp:276]: It's simple: A function that takes `QPointF` can also take `QPoint`. There's nothing wrong with that.

Comment (tsujan) [reply lxqtrotatedwidget.cpp:276]: > And I could not find another call at adjustedPoint() in all my lxqt repos.

OK, if that's the case, it can remain `QPointF`, as you made it here. Let me check it first.

Comment (tsujan) [reply lxqtrotatedwidget.cpp:276]: Sorry, I leave this to other LXQt devs because it's very difficult for me to check *every* LXQt component that depends on `liblxqt` to see whether this change breaks them or not.

Comment (adjamhub) [reply lxqtrotatedwidget.cpp:276]: > It's simple: A function that takes `QPointF` can also take `QPoint`. There's nothing wrong with that.

Yes, that's easy and sure. But the question is: is a QPoint everytime enough if it calls for a QPointF?

Comment (adjamhub) [reply lxqtrotatedwidget.cpp:276]: > Sorry, I leave this to other LXQt devs because it's very difficult for me to check _every_ LXQt component that depends on `liblxqt` to see whether this change breaks it or not.

Just search for "org:lxqt adjustedPoint in:file" in the searchbar?

Comment (tsujan) [reply lxqtrotatedwidget.cpp:276]: > But the question is: is a QPoint everytime enough if it calls for a QPointF?

Yes, it is enough for us.

> Just search for "org:lxqt adjustedPoint in:file" in the searchbar?

I wouldn't trust GitHub for that but OK, I'm merging it. Please be more cautious about changing public methods! Thanks.

**Review by tsujan — APPROVED** [review]
GTM.

Reminder: Test all components for at least one week, before the next release!

**Pull request merged**
